test/parallel/test-tls-client-verify.js

```diff
@@ -71,45 +71,52 @@ function testServers(index, servers, clientOptions, cb) {
     serverOptions.cert = loadPEM(serverOptions.cert);
   }
 
-  var server = tls.createServer(serverOptions, function(s) {
+  const server = tls.createServer(serverOptions, common.mustCall(function(s) {
     s.end('hello world\n');
-  });
+  }));
 
-  server.listen(0, function() {
-    var b = '';
+  server.listen(0, common.mustCall(function() {
+    let b = '';
 
     console.error('connecting...');
     clientOptions.port = this.address().port;
-    var client = tls.connect(clientOptions, function() {
-      var authorized = client.authorized ||
-                       hosterr.test(client.authorizationError);
+    const client = tls.connect(clientOptions, common.mustCall(function() {
+      const authorized = client.authorized ||
+                         hosterr.test(client.authorizationError);
 
       console.error('expected: ' + ok + ' authed: ' + authorized);
 
-      assert.equal(ok, authorized);
+      assert.strictEqual(ok, authorized);
       server.close();
-    });
+    }));
 
     client.on('data', function(d) {
       b += d.toString();
     });
 
-    client.on('end', function() {
-      assert.equal('hello world\n', b);
-    });
+    client.on('end', common.mustCall(function() {
+      assert.strictEqual('hello world\n', b);
+    }));
 
-    client.on('close', function() {
+    client.on('close', common.mustCall(function() {
       testServers(index + 1, servers, clientOptions, cb);
-    });
-  });
+    }));
+  }));
 }
 
 
 function runTest(testIndex) {
-  var tcase = testCases[testIndex];
-  if (!tcase) return;
+  const tcase = testCases[testIndex];
+  if (!tcase) {
+    common.mustCall(function() {
```

Comment:
This doesn't look right. If you put `common.mustCall()` inside a conditional, then the function won't be required to be called unless the conditional is true. But this is an immediately-invoked function, so the `common.mustCall()` seemingly serves no purpose here. Maybe what is meant is for `thisMustRunOnce = common.mustCall(function() {...});` to be outside the conditional so that it always runs, and then inside the conditional: `thisMustRunOnce()`.